sg_otio/cut_diff.py

```diff
@@ -338,7 +338,7 @@ def _check_and_set_changes(self):
         # We have both a Shot and a current clip.
         if self.sg_shot_status and self.sg_shot_status.get("code"):
             # TODO: check if matching by short code is right? #9320
```

Comment:
remove this, no?

Reply:
Let's keep it and remove when we do 9320 if you don't mind.